WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -27,26 +26,34 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 	{
 		TransactionResult = transactionResult;
 
-		var totalAmount = transactionResult.CalculateDestinationAmount();
+		var totalAmount = GetAmount();
 		var total = totalAmount.ToDecimal(MoneyUnit.BTC);
-
 		var fiatTotal = total * fiatExchangeRate;
-
-		_amountFiat = fiatTotal;
-
 		var fiatOriginal = originalAmount * fiatExchangeRate;
 		var fiatDifference = fiatTotal - fiatOriginal;
 
-		_differenceFiat = (fiatDifference > 0
-				? $"{fiatDifference.ToUsd()} More"
-				: $"{Math.Abs(fiatDifference).ToUsd()} Less")
-			.Replace("(", "").Replace(")", "");
-
+		_amountFiat = fiatTotal;
+		_differenceFiatText = GetDifferenceFiatText(fiatDifference);
 		_amount = $"{totalAmount.ToFormattedString()} BTC";
 	}
 
 	public BuildTransactionResult TransactionResult { get; }
 
+	public Money GetAmount()
+	{
+		return TransactionResult.CalculateDestinationAmount();
+	}
+
+	private string GetDifferenceFiatText(decimal fiatDifference)
+	{
+		return fiatDifference switch
+		{
+			> 0 => $"{fiatDifference.ToUsd()} More",
+			< 0 => $"{Math.Abs(fiatDifference).ToUsd()} Less",
+			_ => "Exact Amount"
```

Comment:
```suggestion
			_ => "Same Amount"
```

Reply:
Fight it down https://github.com/zkSNACKs/WalletWasabi/pull/10332#discussion_r1144635325

Reply:
![image](https://user-images.githubusercontent.com/2297442/228510159-3a20bccc-5e79-46d2-90e5-a901a9233b1f.png)

![image](https://user-images.githubusercontent.com/2297442/228510195-a8782ba2-dca2-46a3-8fda-262b1af5a3ed.png)

Reply:
@yahiheb Which one do you prefer?

Reply:
I vote for `same amount` because we are comparing the original amount with the suggested one.